pkgs/development/python-modules/pymoo/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, autograd
+, cma
+, matplotlib
+, numpy
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "pymoo";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "anyoptimization";
+    repo = "pymoo";
+    rev = version;
+    sha256 = "sha256-uS/gRHI8JrXspkms2hlIQNtMiF6N/pMkVmtBzZTMJ2Q=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "cma==2.7" "cma"
+    sed -i pymoo/experimental/constr.py \
+      -e '1ifrom pymoo.core.replacement import ReplacementSurvival'
```

Comment:
not a blocker for this PR, but I feel like this should be upstreamed

Reply:
fixed in 0.6.0